### PR TITLE
feat: add monthly expenditure timegpt tutorial

### PR DESCRIPTION
**Pull request by FedericoGarza**
## Description

This PR includes an example of how to use TimeGPT to forecast monthly expenditures.
Here's an output of the forecasts in mindsdb cloud (locally).

image

## Type of change
- [x] ⚡ New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation, or created issues to update them.
- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

File: docs/sql/tutorials/monthly-expediture-timegpt.mdx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+### Creating an ML Engine
+
+You can check the available engines with this command:
+
+```sql
+SHOW ML_ENGINES;
+```
+
+If you see the TimeGPT engine on the list, you are ready to follow the tutorials.
+
```
Comment (paxcema) [inline docs/sql/tutorials/monthly-expediture-timegpt.mdx:27]: We should probably add this, or something similar:

> If you do not see TimeGPT on the list, you will have to create an instance of the engine first with this command:

> `CREATE ML_ENGINE timegpt FROM timegpt USING api_key = '...'`

> Notice that the USING clause is optional, but you will need to pass an API key eventually (either at model creation, engine creation, model usage, or in the mindsdb configuration file). The only exception to this is when using cloud.mindsdb.com.

Comment (FedericoGarza) [reply docs/sql/tutorials/monthly-expediture-timegpt.mdx:34]: Thank you @paxcema! Fixed in c02d9034dc0a4445a5ec67a466f5c646828293b7.

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Review by paxcema — APPROVED** [review]

**Pull request merged**